frontend/src/components/Admin/AdminCandidateDecider/AdminCandidateDecider.tsx

```diff
@@ -87,6 +88,12 @@ const CandidateDeciderInstanceCreator = ({
     CandidateDeciderAPI.createNewInstance(instance)
       .then((newInstance) => setInstances((instances) => [...instances, newInstance]))
       .then(() => setSuccess(true));
+    setName('');
+    setHeaders([]);
+    setResponses([[]]);
+    setAuthorizedMembers([]);
+    setAuthorizedRoles([]);
+    setFileInKey(Date.now().toString());
```

Comment:
What is this for?

Reply:
Oh, not sure how some code got deleted - just added it back. The purpose of ```setFileInKey``` is to change the ```key``` of the file input component so that after submission, it is forced to re-render. Does this make sense?

Reply:
Ohhh i see i see